src/pages/NewBeer.jsx

```diff
@@ -0,0 +1,69 @@
+import React, {useState} from 'react'
+import NavBar from "../components/NavBar";
+
+function NewBeer({addNewBeer}) {
+  const [name, setName] = useState("")
+  const [tagline, setTagline] = useState("")
+  const [description, setDescription] = useState("")
+  const [firstBrewed, setFirstBrewed] = useState(0)
+  const [brewersTips, setBrewersTips] = useState("")
+  const [attenuation, setAttenuation] = useState(0)
+  const [contributedBy, setContributedBy] = useState("")
+
+  const handleName = (e) => setName(e.target.value)
+  const handleTagline = (e) => setTagline(e.target.value)
+  const handleDescription = (e) => setDescription(e.target.value)
+  const handleFirstBrewed = (e) => setFirstBrewed(Number(e.target.value))
+  const handleBrewersTips = (e) => setBrewersTips(e.target.value)
+  const handleAttenuation = (e) => setAttenuation(Number(e.target.value))
+  const handleContributedBy = (e) => setContributedBy(e.target.value)
+
+  const handleSubmit = (e) => {
+    e.preventDefaul()
+
+    const newBeer = {
+      name: name,
+      tagline:tagline,
+      description: description,
+      firstBrewed: firstBrewed,
+      brewersTips: brewersTips,
+      attenuation: attenuation,
+      contributedBy: contributedBy
+    }
+    addNewBeer(newBeer)
```

Comment:
You forgot to make the post request when creating the beer. By that, it's not creating at all!

`axios.post("https://ih-beers-api2.herokuapp.com/beers/new", newBeer)`